src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+// Actions
+const STATUS_ASKED = 'STATUS_ASKED';
```

Comment:
- [ ] Again, let's make sure that the value of this action type should follow a duck pattern hence it will be something like: 

`const CHECK_STATUS = bookstore/categories/CHECK_STATUS` for reasons mentioned above